packages/astro/src/cli/add/index.ts

```diff
@@ -80,6 +80,17 @@ export default async function seed() {
 	// TODO
 }
 `,
+	CLOUDFLARE_WRANGLER_CONFIG: (name: string) => `\
+{
+	"main": "dist/_worker.js/index.js",
+	"name": ${JSON.stringify(name)},
+  "compatibility_date": ${JSON.stringify(new Date().toISOString().slice(0,10))},
+	"assets": {
+		"binding": "ASSETS",
+		"directory": "./dist"
+	}
```

Comment:
```suggestion
	},
	"observability": {
    "enabled": true
  }
```
from wrangler cli and our deploy guide 

https://github.com/cloudflare/workers-sdk/blob/ca6c01017ccc39671e8724a6b9a5aa37a5e07e57/packages/create-cloudflare/templates/astro/workers/templates/ts/wrangler.jsonc#L13C1-L15C5